sample/Example/Program.cs

```diff
@@ -9,6 +10,14 @@ static class Program
 
         public static void Main()
         {
+            using var listener = new ActivityListener();
+            listener.ShouldListenTo = _ => true;
+            listener.Sample = delegate { return ActivitySamplingResult.AllData; };
+            ActivitySource.AddActivityListener(listener);
+
+            var source = new ActivitySource("Example");
+            using var activity = source.StartActivity();
+            
```

Comment:
This would be removed before merging, but is useful for exploration/smoke testing.